dipy/workflows/reconst.py

```diff
@@ -21,6 +21,195 @@
 from dipy.reconst.shm import CsaOdfModel
 from dipy.workflows.workflow import Workflow
 
+from dipy.reconst import mapmri
+
+
+class ReconstMAPMRIFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'mapmri'
+
+    def run(self, data_file, data_bvecs, data_bvals, small_delta, big_delta,
```

Comment:
Hi @aarya22, can you switch `data_bvecs` and `data_bvals`.  When I run the workflows, I get an error because it does not match with the line [114](https://github.com/nipy/dipy/pull/1395/files#diff-678dce6086d38985220e2ded56251680R114) where `bval` is first and `bvect` the second parameter.
Morever, it keeps the consistency with the others workflows.